### ISSUE TITLE
app: recipe, models tab (part 2)

### ISSUE THREAD

**Issue by gionatamettifogo** [analitico/analitico#66]
top half model details:
- info panel with model metrics
- metrics depend on algorithm type
- feature important with horizontal bars 
- confusion matrix
- graph with loss and training loss on epochs
- various metrics as numbers with explanations

bottom half models list:
- selecting model shows detail on top
- delete model with X icon
- deploy to new endpoint
- link to existing endpoint details
- select/compare models (comparative shopping)
